### PR TITLE
feat: basic layout and header, files from v1

### PR DESCRIPTION
**Pull request by slothtoday**
- Some basic additions of layout and theme for the main page.
- Brought in the index.tsx file from v1 with typescript fixes and corresponding helpers, utils.

Screenshot 2022-02-23 at 12 03 07 AM

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -28,6 +28,9 @@
     ]
   },
   "dependencies": {
+    "@apollo/client": "3.5.9",
+    "@emotion/react": "11.7.1",
+    "@emotion/styled": "11.6.0",
```
Comment (Srile) [inline package.json:33]: Why are we using emotion? Isn't styling handled completely by mui?

**Comment by Srile** [comment]
Left one comment, everything else LGTM

File: package.json (hunk 1 above)
Comment (JSeam2) [reply package.json:33]: Mui v5 uses emotion by default
https://mui.com/getting-started/installation/

File: src/lib/queries.ts [hunk 2]
```diff
@@ -0,0 +1,336 @@
...
+  query GetOneCampaign(
+    $id: String!,
+  ) {
+    buyerCampaign(id: $id) {
+      ${BuyerCampaignFullData}
+    }
+  }
+`;
+
+const HTLCFullData = `
```
Comment (JSeam2) [inline src/lib/queries.ts:263]: The section on HTLC can be deprecated as they are no longer used

File: src/components/Logo.tsx [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+  color: theme.palette.text.primary,
+  fontSize: 45,
+  fontWeight: 600,
+  marginLeft: 8,
+  alignItems: `center`,
+}));
+
+const Logo = () => (
+  <Grid item flexDirection="row" display="flex" alignItems="center">
+    <Image src="/logo.png" alt="logo" width={40} height={40} />
```
Comment (JSeam2) [inline src/components/Logo.tsx:16]: I think we're using Misto's Logo this time round

File: .env.development [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+NEXT_PUBLIC_THE_GRAPH_URI_MATIC=https://api.thegraph.com/subgraphs/name/zestymarket/zesty-market-graph-matic
+NEXT_PUBLIC_THE_GRAPH_URI_MAINNET=https://api.thegraph.com/subgraphs/name/zestymarket/zesty-market-graph-mainnet
+NEXT_PUBLIC_ZESTY_NODE=https://node-1.zesty.market
+NEXT_PUBLIC_IPFS_PINNER=https://pinata.zesty.market
+NEXT_PUBLIC_IPFS_GATEWAY=https://ipfs.zesty.market
+NEXT_PUBLIC_DEFAULT_AD_SPACE_IMAGE=https://ipfs.fleek.co/ipfs/QmPAZBYDYkHntYNNhx9HmS1KCzMg9tzaf2fxZH7KTHjtRd/assets/zesty-market-ad.png
+NEXT_PUBLIC_MARKET_URI=http://localhost:3000
+NEXT_PUBLIC_INFURA_PROJECT_ID=8fa25452dedb4b4c8062329bacf5269b
+NEXT_PUBLIC_BOB=http://localhost:8000
+NEXT_PUBLIC_BEACON_URI=https://beacon.zesty.market
```
Comment (Srile) [inline .env.development:11]: Please replace this with `NEXT_PUBLIC_BEACON_URI=https://beacon2.zesty.market/zgraphql
`

**Comment by Srile** [comment]
LGTM

**Pull request merged**